change/@microsoft-teams-js-0eab053d-5d3a-441c-99f2-b63a8fd47f90.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Removed cardActionsConfig property from externalAppCardActions.processActionSubmit API",
```

Comment:
Need backticks for cardActionsConfig and externalAppCardActions.processActionSubmit API